apps/web/src/pages/templates/WorkflowListPage.tsx

```diff
@@ -199,15 +205,28 @@ function WorkflowListPage() {
     });
   };
 
-  const handleCreateDigestDemoWorkflow = () => {
-    segment.track(TemplateAnalyticsEnum.TRY_DIGEST_CLICK);
-    createDigestDemoWorkflow();
-  };
-
   function onRowClick(row) {
     navigate(parseUrl(ROUTES.WORKFLOWS_EDIT_TEMPLATEID, { templateId: row.values._id }));
   }
 
+  const debouncedSearchChange = useDebounce((value: string) => setSearch(value), 2000);
+
+  const onSearchClearClick = () => {
+    debouncedSearchChange.cancel();
+    setSearchValue('');
+    setSearch('');
+  };
```

Comment:
on input clear button click - cancel the last debounced request and update value and make request right away